packages/rsactftool/PKGBUILD

```diff
@@ -9,8 +9,8 @@ groups=('blackarch' 'blackarch-crypto' 'blackarch-cracker')
 arch=('any')
 url='https://github.com/Ganapati/RsaCtfTool'
 license=('custom:unknown')
-depends=('python2' 'python2-sympy' 'python2-argparse' 'python2-gmpy'
-         'python2-libnum')
+depends=('python' 'python-sympy' 'python-gmpy2'
+         'python-requests' 'python-pycrypto')
```

Comment:
Change `python-pycrypto` for `python-pycryptodome`, it provides the Crypto module.

Reply:
I didn't know pycryptodome was providing a different implementation of this class.